install.sh

```diff
@@ -0,0 +1,152 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+cat <<EOF
+     _           _   _   _      
+ ___| |__  _   _| |_| |_| | ___ 
+/ __| '_ \\| | | | __| __| |/ _ \\
+\__ \\ | | | |_| | |_| |_| |  __/
+|___/_| |_|\\__,_|\\__|\\__|_|\\___|
+                                
+https://www.shuttle.rs
+https://github.com/shuttle-hq/shuttle
+
+Please file an issue if you encounter any problems!
+===================================================
+EOF
+
+if ! command -v curl &>/dev/null; then
+  echo "curl not installed. Please install curl."
+  exit
+fi
+
```

Comment:
Check if `cargo-shuttle` is already install and if it is already the latest version.

Reply:
a27d05534805febfab302fdcac76f25f9b79d4bb